i18n/es_ES.json

```diff
@@ -0,0 +1,253 @@
+{
+"%s - Picture #%s": "%s - Imagen #%s",
+"%s Item(s) In Cart"": "%s Artículo (s) en el carrito",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected character ('"' (code 34)): was expecting a colon to separate field name and value](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4644676)